src/twisted/logger/test/test_format.py

```diff
@@ -36,6 +36,12 @@ class FormattingTests(unittest.TestCase):
     Tests for basic event formatting functions.
     """
 
+    def format(self, logFormat: AnyStr, **event: object) -> str:
+        event["log_format"] = logFormat
+        result = formatEvent(event)
+        self.assertIs(type(result), str)
+        return result
```

Comment:
Maybe add a short docstring with the purpose of this function.


Also, minor comment, since we are here, maybe we can have `assertion(EXPECTED, ACTUAL)` as a convention for the assertions.

```suggestion
    def format(self, logFormat: AnyStr, **event: object) -> str:
        """
        Return the text rendering of C{event} based on the
        C{logFormat} template.
        
        This is a wrapper for L{format} to reduce code
        duplication in these tests.
        """
        event["log_format"] = logFormat
        result = formatEvent(event)
        self.assertIs(str, type(result))
        return result
```